resources/qml/Sidebar.qml

```diff
@@ -166,15 +166,6 @@ Rectangle
         anchors.topMargin: UM.Theme.getSize("default_margin").height
     }
 
-    currentModeIndex:
-    {
-        var index = parseInt(UM.Preferences.getValue("cura/active_mode"))
-        if(index)
-        {
-            return index;
-        }
-        return 0;
-    }
```

Comment:
Wait, but this is still being used at (new) line 262, where you added additional checks?